### PR TITLE
JBPM-7252 - Upgrade camel version to 2.21.0

### PR DESCRIPTION
**Pull request by tsurdilo**

### CODE REVIEW

**Comment by tsurdilo** [comment]
@mswiderski camel 2.21.0 also needs updated version of commons-net which was done in pom

**Comment by tsurdilo** [comment]
pr for https://github.com/kiegroup/jbpm-work-items/pull/37

**Comment by mswiderski** [comment]
Jenkins execute full downstream build

**Comment by mswiderski** [comment]
@tsurdilo in general looks good though we need to make sure kie-camel works ok afret upgrade too, thus full downstream

**Comment by tsurdilo** [comment]
Jenkins execute full downstream build

**Comment by tsurdilo** [comment]
Jenkins execute full downstream build

**Comment by tsurdilo** [comment]
Jenkins execute full downstream build

**Comment by tsurdilo** [comment]
Jenkins execute full downstream build

**Comment by tsurdilo** [comment]
Jenkins execute full downstream build

**Comment by tsurdilo** [comment]
Jenkins execute full downstream build

**Comment by tsurdilo** [comment]
jenkins build fails because of enforcer rules issue in jbpm - fixed with pr: https://github.com/kiegroup/jbpm/pull/1228

**Comment by tsurdilo** [comment]
Jenkins execute full downstream build

**Comment by tsurdilo** [comment]
Jenkins execute full downstream build

**Comment by mariofusco** [comment]
Jenkins execute full downstream build

**Comment by tsurdilo** [comment]
Jenkins execute full downstream build

**Comment by mariofusco** [comment]
Jenkins execute full downstream build

**Comment by tsurdilo** [comment]
Jenkins execute full downstream build

**Comment by tsurdilo** [comment]
Jenkins execute full downstream build

**Comment by tsurdilo** [comment]
Jenkins execute full downstream build

File: pom.xml [hunk 1]
```diff
@@ -251,6 +251,8 @@
 
     <version.org.quartz-scheduler>2.2.3</version.org.quartz-scheduler>
 
+    <version.org.jboss.spec.javax.ws.rs.jboss-jaxrs-api_2.1_spec>1.0.0.Final</version.org.jboss.spec.javax.ws.rs.jboss-jaxrs-api_2.1_spec>
```
Comment (yurloc) [inline pom.xml]: Shouldn't this be managed by IP BOM?

**Pull request merged**